Harvest.Core/Data/DbInitializer.cs

```diff
@@ -145,7 +145,7 @@ private async Task CreateEquipmentRates(User createdBy)
             rate.Type = Rate.Types.Equipment;
             rate.Description = "PLS Row Planter";
             rate.BillingUnit = "Plant Sciences";
-            rate.Account = "3-APSNFDS";
+            rate.Account = "3-APSNFDS--RAPB";
```

Comment:
It's worth asking Bryan about the preferred format here because I've seen it written where if you don't have a subacct you have to put in 5 dashes (or however long the subacct is).  If that's the case our parser would have to handle that.